lib/active_admin/inputs/filters/date_picker_input.rb

```diff
@@ -0,0 +1,15 @@
+module ActiveAdmin
+  module Inputs
+    module Filters
+      class DatePickerInput < ::Formtastic::Inputs::DatePickerInput
+        include Base
+
+        def input_html_options
+          {
+            class: 'datepicker'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.